packages/database/e2e/DatabaseStatics.e2e.js

```diff
@@ -15,7 +15,13 @@
  *
  */
 
+const { PATH, wipe } = require('../helpers');
```

Comment:
```suggestion
const { PATH, wipe } = require('./helpers');
```

In same directory - import failing on CI